WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -86,6 +87,7 @@
                   <%= registered_date %>
                 </span>
               </td>
+              <td class="events"><%= registration.events.length() %></td>
```

Comment:
Nitpick: for methods without parameters, you can omit the parenthesis.

Very much not a nitpick: while definitely working and doing what is expected, I suspect this generates an n+1 query, because we don't always eager load registration's events.
In other words, each registration's line will fire an SQL select to get all the registration's events, which would not be nice when the competition has a lot of registrations.
You can check this behavior in the server's log: start your server, open the registrations page for like wc2017, and you should see a whole lot of selects.
Luckily the fix is easy, as we do eager load the registration's events when displaying them.
You just need to remove [this line](https://github.com/thewca/worldcubeassociation.org/blob/b4386218588b3f6ba954418d2240cdaf65cb5883/WcaOnRails/app/controllers/registrations_controller.rb#L43), and change [this one](https://github.com/thewca/worldcubeassociation.org/blob/b4386218588b3f6ba954418d2240cdaf65cb5883/WcaOnRails/app/controllers/registrations_controller.rb#L42) to:
`@registrations = @competition.registrations.includes(:user, :registration_payments, :events)`

Then reload the page and instead of ~1000 small selects, you should see just one.